cli/tests/integration/watcher_tests.rs

```diff
@@ -1122,6 +1122,46 @@ fn test_watch_unload_handler_error_on_drop() {
   check_alive_then_kill(child);
 }
 
+#[test]
+fn run_watch_blob_urls_reset() {
+  let _g = util::http_server();
```

Comment:
Nitpick: If willing, could you update this test to use the new `TestContextBuilder`? I know a bunch of the tests here don't use it, but we're slowly transitioning over.

Reply:
Actually, maybe it's not easy to do the `child_lines` thing with it atm.